static.go

```diff
@@ -237,6 +240,37 @@ const (
 				</form>
 			</div>
 		</section>
+		<section class="row">
+			<div id="restore" class="col-md-6">
+				<h3>Restore</h3>
```

Comment:
I didn't think so, particularly since the functionality will be limited to
a particular set of users that will know this distinction.